website/docs/blog.mdx

```diff
@@ -328,7 +328,7 @@ authors:
 The configuration file can be localized, just create a localized copy of it at:
 
 ```bash
-website/i18n/<locale>/docusaurus-plugin-content-blog/authors.yml
```

Comment:
this is a syntax we use on quite a few places, changing it only there makes it inconsistent.

We should either change it everywhere, or nowhere.

Reply:
agree for normalization. I don't have a strong opinion on which syntax to use